<launch>
  #<arg name="model" />
  <arg name="gui" default="True" />

  #<param name="robot_description" textfile="$(find orion5)/urdf/orion5.urdf" />
  <param name="_use_gui" value="$(arg gui)"/>

  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
  <node name="orion5_joint_listener" pkg="orion5" type="orion5_joint_listener.py"/>
  
</launch>
